README.md

```diff
@@ -67,15 +77,41 @@ optional arguments:
                         module homepage
   -l LICENSE, --license LICENSE
                         license
-  -ex EXTRA_FILES, --extras EXTRA_FILES
-                        extra files
   -c COMMAND_LINE_ARGS, --cmdargs COMMAND_LINE_ARGS
                         module command line arguments
   -r REDIS_MIN_VERSION, --redis-min-version REDIS_MIN_VERSION
                         redis minimum version
   -rl RLEC_MIN_VERSION, --rlec-min-version RLEC_MIN_VERSION
                         rlec minimum version
   -O OS, --os OS        Build target OS (Darwin/Linux)
+
+  Next to follow are a set of flags specifing your module's capabilities.
+
+  -crr, --cap-reshard-rebalance
```

Comment:
There should not be specific flags for each capability, just `--capabilities foo,bar,baz`